disnake/utils.py

```diff
@@ -316,6 +317,32 @@ def oauth_url(
     return url
 
 
+def parse_token(token: str) -> Tuple[int, datetime.datetime, bytes]:
+    """Parse a token into its parts
+    
+    Returns 
+    
+    Parameters
+    -----------
+    token: :class:`token`
+        The bot token
+    
+    Returns
+    --------
+    Tuple[:class:`int`, :class:`datetime.datetime`, :class:`bytes`]
+        the bot's id, the time when the token was generated and the hmac.
+    """
+    parts = token.split('.')
+    
+    user_id = int(b64decode(parts[0]))
+    
+    timestamp = int.from_bytes(b64decode(parts[1] + '=='), 'big')
```

Comment:
```suggestion
    timestamp = int.from_bytes(b64decode(parts[1] + '=='), 'big')
    timestamp += 1293840000  # Epoch used for tokens
```

Reply:
Actually ain't a thing, the image seems to be outdated cause it works just fine without an epoch, idk why tho.